src/App.js

```diff
@@ -1,13 +1,11 @@
-import logo from "./logo.svg";
+import React, { useState } from "react";
 import "./App.css";
-
 function App() {
+  const [count, setCount] = useState(0);
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>Learn react in the gym</p>
-      </header>
+    <div>
+      <p>Button has been clicked: {count} times</p>
+      <button onClick={() => setCount(count + 1)}>Click ME</button>
```

Comment:
It better for this approach to use the previous value.
```suggestion
      <button onClick={() => setCount((prev) => prev + 1)}>Click ME</button>
```